oauth2-server/src/main/java/com/clouway/oauth2/PublicKeys.java

```diff
@@ -0,0 +1,25 @@
+package com.clouway.oauth2;
+
+import com.google.common.base.Optional;
+
+import java.security.Key;
+import java.util.Map;
+
+/**
+ * @author Ianislav Nachev <qnislav.nachev@clouway.com>
+ */
+public interface PublicKeys {
```

Comment:
Add comments to this public interface, as it's going to be used by client code which need to know what's the purpose of it.